env.sh

```diff
@@ -2,8 +2,8 @@
 
 export MACOSX_DEPLOYMENT_TARGET=10.10
 
-export BRAVE_TOR_VERSION="1"
-export TOR_VERSION="0.4.5.10"
+export BRAVE_TOR_VERSION="0"
```

Comment:
I see BRAVE_TOR_VERSION used in the output binary, what does this field mean?
And why does it go from 1 to 0?

Reply:
For tor-0.4.5.10, we pushed updated twice. 1. when the 0.4.5.10 was released by the tor project 2. to update the underlying OpenSSL library which had high severity security issues. 

We use the `BRAVE_TOR_VERSION` to track multiple releases of the same tor client version.

Reply:
Given that this is the first release for `0.4.5.11` we bumped the number back to 0